package.json

```diff
@@ -110,6 +110,11 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Enable/disable Unibeautify formatter"
+                },
+                "unibeautify.defaultConfig": {
+                    "type": "string",
+                    "default": null,
+                    "description": "Absolute path to your .unibeautifyrc file outside the workspace folder. This will override the workspace's own .unibeautifyrc config file."
```

Comment:
> This will override the workspace's own .unibeautifyrc config file.

Will it? Or is this only the default *when `.unibeautifyrc` config file **does not exist within the workspace**?*

Looking at https://github.com/muuvmuuv/vscode/blob/a0c792544f8df0db4853ac9008d87f719b2536bf/src/EditProvider.ts#L121-L134 this description is accurate. However, I think this is not actually what we want.

@muuvmuuv I believe your use case was you did *not* have a `.unibeautifyrc` file within the workspace. You wanted to fallback on a `defaultConfig` in those cases. By adding `defaultConfig` for VSCode this allows you to configure within your `User Settings` of VSCode.

However, with the currently implementation, if you set `defaultConfig` in your `User Settings` (instead of `Workspace Settings`) then it would actually ignore the `.unibeautifyrc` within the workspace! This is not what we want.

cc @szeck87  right?

@muuvmuuv Could you swap around the login at https://github.com/muuvmuuv/vscode/blob/a0c792544f8df0db4853ac9008d87f719b2536bf/src/EditProvider.ts#L121-L134 such that the `defaultConfig` is a *fallback for when `.unibeautifyrc` is not found within the workspace? Thank you!

Reply:
makes sense. I‘ll change this.